src/Options.cpp

```diff
@@ -223,6 +224,26 @@ bool Options::parseCommandLine(int argc, const char* const* argv)
         "compression",
         po::value<int>(&png_compression_level_)->default_value(-1),
         "PNG compression level: 0 (none) to 9 (best), or -1 (default)"
+    )(
+        "raw-samplerate",
+        po::value<int>(&samplerate_),
+        "samplerate for raw audio input"
+    )(
+        "raw-bits-per-sample",
```

Comment:
This is functionally the same, so really a personal preference - but I think I'd like to have a `--raw-format` option, as in https://github.com/bbc/audiowaveform/issues/70#issuecomment-440011636 to set the bits per sample, encoding, endianness. We can store this as a string and pass it directly to `configure()`, which would then parse and validate it. But feel free to leave this if you don't want to make the change.

Reply:
done